parcel-resolver-veda/index.d.ts

```diff
@@ -178,6 +178,7 @@ export interface LayerInfo {
    */
   export interface DatasetData {
     featured?: boolean;
+    sourceExclusive?: string;
```

Comment:
If this is a binary value, would it make sense to make it boolean?

Reply:
I see the value is actually used, so it is not a boolean flag.

https://github.com/NASA-IMPACT/veda-ui/pull/913/files#diff-e7c2bc61a035bdafe3d86d260f7cddb201aadb0195fcf1cd261139932ea00cc4R122

So, basically, you define here, which source value can exclusively be used. That is a bit redundant, though, because the is already defined under `taxonomy`, so a boolean flag would be cleaner for this simple case where only the source of the current dataset can be exclusive.

Reply:
I think I did this because some datasets look like they can have more than 1 source value like [here](https://raw.githubusercontent.com/US-GHG-Center/veda-config-ghg/develop/datasets/oco2-mip-co2budget-yeargrid-v1.data.mdx). So for me to set the filter the easiest, i opted to do it like so. lmk if there are better ideas though and I will go change 🏃🏼‍♀️

Reply:
This is great for now, thanks! We can refine later, if we learn more about the use cases and needs for these restrictions - or find out we can avoid them with other solutions.